smart_open/tests/test_smart_open.py

```diff
@@ -1163,6 +1163,19 @@ def test_write_text_gzip(self):
             actual = fin.read()
         self.assertEqual(text, actual)
 
+class HostNameTest(unittest.TestCase):
+    def test_host_name(self):
+        host = 'http://a.com/b'
+        expected = 'http://a.com/b'
+        self.assertTrue(expected == smart_open_lib._add_sheme_to_host(host))
```

Comment:
It's better to store the result of the function in a separate variable use .assertEquals(expected, actual).

That way, if the assert fails, the test runner will show you both the expected and actual values, and how they differ. That will save you time debugging.

If you just use .assertTrue, the test runner will just show you when that value is False, and you'll have to debug your code to work out how exactly the comparison failed.